tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable for-direction */
+//  I modified slightly my code used in previous exercises.
+
+export const numPrimorial = ( num ) => {
+  let primesArray = [];
+  for ( let i = 2; i > 0; i ++) {
+    if ( primesArray.length + 1 <= num ) {
+      getPrime( i, primesArray );
```

Comment:
probably it would be better if you not change original isPrime function, and add values to array in case this function return true, as it is not very good when your function has side effects like this

also while loop can be used here, in condition you will have code which you have in `if` statement and until primesArray will smaller than num you will run loop

Reply:
Done